lib/presentation/view/header/context_menu_header_builder.dart

```diff
@@ -40,18 +40,21 @@ class ContextMenuHeaderBuilder {
 
   ContextMenuHeaderBuilder(this._key, this._file);
 
-  ListTile build() {
-    return ListTile(
-      key: _key,
-      leading: _file.fileIcon(),
-      title: Transform(
-        transform: Matrix4.translationValues(-16, -2, 0.0),
-        child: Text(
-          _file.fileName(),
-          maxLines: 1,
-          style: TextStyle(fontSize: 14, color: AppColor.documentNameItemTextColor),
+  Transform build() {
+    return Transform.translate(
+        offset: Offset(0,3.5),
+        child: ListTile(
+          key: _key,
+          leading: Transform.translate(
+              offset: _file.fileIconImagePath() == 'assets/images/ic_folder.svg' ? Offset(0,0) : Offset(0,2),
```

Comment:
Use constants instead please and I think you `fileIconImagePath` is not needed

Reply:
You could do : `_file is WorkGroupFolderPresentationFile ? ..` or smth like this wdyt ?

Reply:
It was so good that I couldn't think of it. Thank you very much